addons/barcodes/static/src/float_scannable_field.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+
+    <t t-name="barcodes.FloatScannableField" t-inherit="web.FloatField" t-inherit-mode="primary" owl="1">
+        <xpath expr="//input" position="attributes">
+            <attribute name="t-on-barcode_scanned">onBarcodeScanned</attribute>
+            <attribute name="t-att-data-enable-barcode">true</attribute>
```

Comment:
I decided to encode the `enableBarcode` data in the template, instead of doing it via `useEffect`. What do you think?

Reply:
since you have the template, it makes completely sense. Also, it's slightly better, performance wise :)